files/en-us/glossary/clickjacking/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Clickjacking
+slug: Glossary/Clickjacking
+tags:
+  - Clickjacking
+  - Interface-based attack
+  - Glossary
+  - Security
+  - vulnerability
+  - exploit
+---
+<p>Clickjacking is an interface-based attack that tricks website users into unwarily clicking on malicious links. In clickjacking, the attackers embed their malicious links into buttons or legitimate pages in a website. In an infected {{glossary("Site")}}, whenever a user clicks on a legitimate link, the attacker gets the confidential information of that user - which ultimately compromises the user's privacy on the Internet. Clickjacking falls under the <a href="https://www.owasp.org/index.php/Top_10-2017_A6-Security_Misconfiguration">A6 – Security Misconfiguration</a> item in OWASP’s 2017 Top 10 list.</p> 
```

Comment:
A couple of suggestions here:

- Punctuation fix
- Dropping the OWASP citation here (I thinking citing a four-year old list undermines the point)

```suggestion
<p>Clickjacking is an interface-based attack that tricks website users into unwarily clicking on malicious links. In clickjacking, the attackers embed their malicious links into buttons or legitimate pages in a website. In an infected {{glossary("Site")}}, whenever a user clicks on a legitimate link, the attacker gets the confidential information of that user, which ultimately compromises the user's privacy on the Internet.</p> 
```